ocs_ci/ocs/ocp.py

```diff
@@ -563,8 +563,10 @@ def wait_for_resource(
                     actual_status = status
                     if error_condition is not None and status == error_condition:
                         raise ResourceInUnexpectedState(
-                            f"Status of '{resource_name}' at column {column}"
-                            f" is {status}."
+                            resource_name,
```

Comment:
We are raising `ResourceInUnexpectedState` here (and below), yet the Exception you've made changes to is `ResourceWrongStatusException`. Am I missing something here or should we be changing `ResourceInUnexpectedState` instead?

Reply:
Nope, good catch! I had conflated the two. I'll push a separate commit to add similar changes to `ResourceInUnexpectedState`.

Reply:
Actually, @clacroix12 - now that I look at this more closely, these exceptions really are the same concept. `ResourceInUnexpectedState` is currently just the base `Exception` by another name. I could make it inherit from `ResourceWrongStatusException`, or consolidate the codebase to use that instead of its less-informative cousin. Thoughts?

Reply:
I _think_ it's best to move to a single "resource is in the wrong state/status" exception, so I'm testing a commit which does that. I can push it here if it works out and you like the idea.

Reply:
Yeah, from the names they definitely seem like they describe the same error so I'm personally in favor of consolidating them.

Pinging @ebenahar and @petr-balogh as they are the authors of those exceptions according to `git blame` just in case they have any objections.

Reply:
Okay - just pushed another commit that does that. Tested with this build https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/job/ocs-ci-kvm/231/ (an extremely rare green Power build!)